migrations/versions/aaa4b447e0c5_add_discounted_amount_field_to_price_.py

```diff
@@ -24,7 +24,7 @@ def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column(
         "price",
-        sa.Column("discounted_amount", sa.FLOAT),
+        sa.Column("discounted_amount", sa.FLOAT, nullable=True),
```

Comment:
`nullable=True` isn't necessary as it is the default behaviour:

> When True, will normally generate nothing...

https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.Column.params.nullable